dotCMS/src/integration-test/java/com/dotcms/auth/providers/saml/v1/SAMLHelperTest.java

```diff
@@ -284,4 +286,59 @@ public void test_getRoleKeySubstitution_processReplacement_doing_substitution()
         Assert.assertEquals(expected, roleResult);
     }
 
+    /**
+     * Method to test: {@link SAMLHelper#createNewUser(User, Attributes, IdentityProviderConfiguration)}
+     * Given Scenario: creates an user only with name id
+     * ExpectedResult: The user is created successfully with random values
+     *
+     */
+    @Test()
+    public void test_createNewUser_non_Attrs() throws NoSuchAlgorithmException {
+
+        final SamlAuthenticationService samlAuthenticationService         = new MockSamlAuthenticationService();
+        final CompanyAPI companyAPI                                       = mock(CompanyAPI.class);
+        final Company    company                                          = new Company();
+        final SAMLHelper           		samlHelper                        = new SAMLHelper(samlAuthenticationService, companyAPI);
+
+        company.setAuthType(Company.AUTH_TYPE_EA);
+        when(companyAPI.getDefaultCompany()).thenReturn(company);
+        final IdentityProviderConfiguration identityProviderConfiguration = mock(IdentityProviderConfiguration.class);
+        final Attributes attrs = new Attributes.Builder().nameID(UUID.randomUUID()).build();
+
+        final User user = samlHelper.createNewUser(APILocator.systemUser(), attrs, identityProviderConfiguration);
+
+        Assert.assertNotNull(user);
+        Assert.assertEquals(samlHelper.hashIt(attrs.getNameID().toString()), user.getUserId());
```

Comment:
should we check here the others attribute?, for example check that the email is really null